tests/UnitTests/CoreGraphics.drawing/CGContextDrawing_FlushTests.cpp

```diff
@@ -0,0 +1,42 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#include "DrawingTest.h"
+
+#if WINOBJC
+#include "CGContextInternal.h"
+DRAW_TEST_F(CGContextFlush, FillFlush, WhiteBackgroundTest<>) {
+    CGContextRef context = GetDrawingContext();
+    CGRect bounds = GetDrawingBounds();
+    _CGContextPushBeginDraw(context);
+
+    CGContextSetRGBFillColor(context, 1, 0, 0, 1);
+    CGContextFillRect(context, bounds);
+
+    CGContextFlush(context);
+
+    BYTE* dataPtr = static_cast<BYTE*>(CGBitmapContextGetData(context));
+    ASSERT_NE(dataPtr, nullptr);
+
+    EXPECT_EQ(dataPtr[0], 00);
+    EXPECT_EQ(dataPtr[1], 00);
+    EXPECT_EQ(dataPtr[2], 0xff);
+    EXPECT_EQ(dataPtr[3], 0xff);
+
+    _CGContextPopEndDraw(context);
```

Comment:
As earlier common indicated, it should be, but with more drawings it can be a valid drawing test.

:D

---
In reply to: [105513897](https://github.com/Microsoft/WinObjC/pull/2208#discussion_r105513897) [](ancestors = 105513897)